addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/XxeScanRule.java

```diff
@@ -191,7 +191,7 @@ public void scan() {
             // using a challenge/response model based on a random string
 
             // Skip XXE Remote File Inclusion Attack when callback extension is not available.
-            if (ChallengeCallbackImplementor.getExtensionCallback() != null) {
+            if (ChallengeCallbackImplementor.getCallbackService() != null) {
```

Comment:
The comment is no longer accurate, it will lead to NPE if the extension is disabled.